README.md

```diff
@@ -439,6 +439,19 @@ kubectl logs kodata
 2018/07/19 23:38:29 Hello there
 ```
 
+## Multi-Platform Images
+
+If `ko` is invoked with `--platform=all`, for any image that it builds that is
+based on a multi-architecture image (e.g., the default
+`gcr.io/distroless/static:nonroot`, `busybox`, `alpine`, etc.), `ko` will
+attempt to build the Go binary using Go's cross-compilation support and produce
+a multi-architecture image, with support for each architecture supported by the
+base image.
+
+If `ko` is invoked with `--platform=<some-platform>` (e.g., `--platform=amd64`
+or `--platform=arm64`), then it will attempt to build an image for that
+architecture only, assuming the base image supports it.
```

Comment:
I'd include that (based on our overarching bias towards Go idioms) the default for this is based on GOOS and GOARCH